model/roles_test.go

```diff
@@ -526,3 +527,176 @@ func TestLoadRoleManifestRunGeneral(t *testing.T) {
 		assert.Nil(err)
 	}
 }
+
+func TestResolveLinks(t *testing.T) {
+	workDir, err := os.Getwd()
+
+	assert.NoError(t, err)
+
+	var releases []*Release
+
+	for _, dirName := range []string{"ntp-release", "tor-boshrelease"} {
+		releasePath := filepath.Join(workDir, "../test-assets", dirName)
+		releasePathBoshCache := filepath.Join(releasePath, "bosh-cache")
+		release, err := NewDevRelease(releasePath, "", "", releasePathBoshCache)
+		if assert.NoError(t, err) {
+			releases = append(releases, release)
+		}
+	}
+
+	roleManifestPath := filepath.Join(workDir, "../test-assets/role-manifests/multiple-good.yml")
+	roleManifest, err := LoadRoleManifest(roleManifestPath, releases)
+	assert.NoError(t, err)
+
+	errList := roleManifest.resolveLinks()
+	assert.Empty(t, errList)
+
+	role := roleManifest.LookupRole("myrole")
+	var job *Job
```

Comment:
Why not

```
job := role.LookupJob("ntpd")
```

Reply:
Because that didn't exist yet when I wrote that line 😊 
Fixed in c854927c6dc05962f2e2c02c4f7ca472eab163bf